### PR TITLE
Add new apps, remove old apps and fix outdated links/tags

### PR DESCRIPTION
**Pull request by DimitrisPa**
### Add / Remove
**Audio**

- Airtime (_Remove/Discontinued_)
- Auryo (_Remove/Discontinued_)
- G4Music
- Gnormalize (_Remove/Discontinued_)
- Lyricfier (_Remove/Discontinued_)
- Spicetify
- Spot
- Spotio (_Remove/Discontinued_)
- Spotube
- Tidal-hifi

**3rd party Client**

- NeoChat
- Tokodon

**E-Book Utilities**

- PDF Slicer

**Editors**

- CudaText

**Engine Re-creations (require the actual game)**

- NXEngine-evo (#445)

**Gaming Applications**

- Wine-tkg
- Wine-wayland

**Screenshot**

- Spectacle

**Streaming**

- Boatswain

**Web Service Client**

- GnomeTwitch (_Remove/Discontinued_)
- Streamlink Twitch GUI

**Compartmentalization**

- Pods
- VMware Workstation Player

**Sharing Files**

- AnyDesk
- Connections

**System Maintenance**

- AppImageLauncher

**VPN** (_New Category_)

- CyberGhost
- ExpressVPN
- IVPN
- Mozilla VPN
- Mullvad
- NordVPN
- OpenVPN
- Private Internet Access
- ProtonVPN
- PureVPN
- Surfshark
- Windscribe
- WireGuard

**Command Line Utilities**

- pfetch
- reddio
- RTV (_Remove/Archived_)
- tldr-pages

**Custom Linux Kernels**

- linux-tkg

**Tiling window managers**

- Bismuth

**Dynamic window managers**

- Kröhnkite

**Reddit**

- Asahi Linux
- Command Line
- Deepin
- Haiku
- Nix
- Tails
- Whonix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1127,6 +1142,21 @@ the WebKit rendering engine.
...
+- ![Nonfree][Money icon][![Open-Source Software][OSS Icon]](https://protonvpn.com/) [ProtonVPN](https://protonvpn.com/) - High-speed Swiss VPN that safeguards your privacy.
+- ![Nonfree][Money icon][PureVPN](https://www.purevpn.com/) - PureVPN is much more than your usual VPN services provider. It offers unparalleled security and anonymity, making it your only choice for a secure online browsing experience.
+- ![Nonfree][Money icon][Surfshark](https://surfshark.com/) - All you need in a VPN and more in one easy-to-use app.
+- ![Nonfree][Money icon][![Open-Source Software][OSS Icon]](https://github.com/Windscribe/Desktop-App) [Windscribe](https://windscribe.net/) - Browse the web privately as it was meant to be.
+- [![Open-Source Software][OSS Icon]](https://www.wireguard.com/repositories/) [WireGuard](https://www.wireguard.com/) - WireGuard is an extremely simple yet fast and modern VPN that utilizes state-of-the-art cryptography. It aims to be faster, simpler, leaner, and more useful than IPsec, while avoiding the massive headache.
```
Comment (luong-komorebi) [inline README.md:1158]: https://tailscale.com/ have you ever tried this ?

**Review by luong-komorebi — APPROVED** [review]
awesome work, as always

File: README.md (hunk 1 above)
Comment (DimitrisPa) [reply README.md:1158]: No i didn't know it, but it's seems to be based on WireGuard and it has very clean UI. I could include it into this PR if you have it try it in the past.

Comment (luong-komorebi) [reply README.md:1158]: ahhh that is not necessary. Since you are recommending wireguard, I thought I just wanna share something nice to you 👍🏻
